legal-api/report-templates/alterationNotice.html

```diff
@@ -0,0 +1,39 @@
+[[macros.html]]
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Alteration Notice</title>
+        <meta charset="UTF-8">
+        <meta name="author" content="BC Registries and Online Services">
+        [[common/style.html]]
+    </head>
+    <body>
+        <div class="header">
+            <table class="header-table" role="presentation">
+                <tr class="no-page-break">
+                    <td>
+                        [[logo.html]]
+                    </td>
+                    <td>
+                        <div class="report-type">ALTERATION NOTICE</div>
+                        <div class="report-type-desc">Business Corporations Act</div>
+                    </td>
+                </tr>
+            </table>
+            <div class="business-name-header">
+                <label class="lbl-business-name">{{business.legalName}}</label>
+            </div>
+        </div>
+        [[common/businessDetails.html]]      
+        <div class="section-header mt-5 pt-4">ALTERATION NOTICE</div>
+        <div class="container pt-4">
+            [[alteration-notice/effectiveDate.html]]
+            [[alteration-notice/businessTypeChange.html]]
+            [[alteration-notice/legalNameChange.html]]
+            [[notice-of-articles/benefitCompanyStmt.html]]
```

Comment:
common components should be moved to common, so that future maintainers know that their changes could affected other filings

Reply:
Fixed.